config/schedule.rb

```diff
@@ -0,0 +1,11 @@
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+#
+# set :output, "/path/to/my/cron_log.log"
+#
+every 2.hours do
+  # command "/usr/bin/some_great_command"
+  # runner "MyModel.some_method"
+  # rake "some:great:rake:task"
+end
```

Comment:
WOW! Was that magic? Lol.

On 4/15/2016 12:12 AM, Brandon wrote:
>
> In config/schedule.rb 
> <https://github.com/Ditofry/colorado-alarm-api/pull/2#discussion_r59830477>:
>
> > @@ -0,0 +1,11 @@
> > +# It's helpful, but not entirely necessary to understand cron before proceeding.
> > +# http://en.wikipedia.org/wiki/Cron
> > +
> > +#
> > +# set :output, "/path/to/my/cron_log.log"
> > +#
> > +every 2.hours do
> > +  # command "/usr/bin/some_great_command"
> > +  # runner "MyModel.some_method"
> > +  # rake "some:great:rake:task"
> > +end
>
> This is why people get addicted to ruby and, specifically, to Ruby on 
> Rails. I don't know if you guys have had to deal with setting up chron 
> jobs before, but this is an INSANELY nice dsl for writing chron jobs. 
> Like I said in the PR description, there will be more added to this 
> after we touch base on the weather api methods that are currently in 
> another branch.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/Ditofry/colorado-alarm-api/pull/2/files/879916121062dcbfe03445a8931e21e9743cb450#r59830477>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194